artemis-console-extension/artemis-extension/src/artemis-extension/artemis/config-manager.ts

```diff
@@ -0,0 +1,63 @@
+import { log } from "./globals"
+
+export type Artemisconfig = {
+    /**
+     * Configuration for branding & styles.
+     */
+    jmx: JMXConfig
+
+}
+
+/**
+ * JMX configuration type.
+ */
+export type JMXConfig = {
+    domain: string
+}
+
+export const ARTEMISCONFIG_JSON = 'artemisconfig.json'
+
+class ConfigManager {
+    private config?: Promise<Artemisconfig>
+
+    getArtemisconfig(): Promise<Artemisconfig> {
+        if (this.config) {
+          return this.config
+        }
+    
+        this.config = this.loadConfig()
+        return this.config
+      }
+    
+
+    private async loadConfig(): Promise<Artemisconfig> {
+    
+        try {
+            const res = await fetch(ARTEMISCONFIG_JSON)
+         if (!res.ok) {
+
+            log.info("Unable to load Artemis Config using Default");
```

Comment:
Indentation seems wonky.

Comma needed to make the log message more understandable: ", using Default"